charts/temporal/templates/admintools-deployment.yaml

```diff
@@ -3,6 +3,10 @@ apiVersion: apps/v1
 kind: Deployment
 metadata:
   name: {{ include "temporal.componentname" (list $ "admintools") }}
+  {{- if or $.Values.additionalAnnotations $.Values.admintools.deploymentAnnotations }}
```

Comment:
Let's remove the gating, we can just allow `annotations:` to be empty, it's valid syntax.

Reply:
Removed the gating.